testing/hyperfine/APKBUILD

```diff
@@ -0,0 +1,26 @@
+# Contributor: Leo <thinkabit.ukim@gmail.com>
+# Maintainer: Leo <thinkabit.ukim@gmail.com>
+pkgname=hyperfine
+pkgver=1.5.0
+pkgrel=0
+pkgdesc="Command-line benchmarking tool"
+url="https://github.com/sharkdp/hyperfine"
+arch="x86_64" # Limited by cargo
+license="Apache-2.0 AND MIT"
+makedepends="cargo"
+source="$pkgname-$pkgver.tar.gz::https://github.com/sharkdp/hyperfine/archive/v$pkgver.tar.gz"
+
+build() {
+	cargo build --release --verbose
+}
+
+check() {
+	cargo test all --release --verbose
+}
+
+package() {
+	cargo install --path . --root="$pkgdir"/usr
```

Comment:
Using root in /usr seems odd here. I am not aware of a better way just now, but would welcome one.